src/widget/systemtrayicon.cpp

```diff
@@ -366,11 +381,15 @@ void SystemTrayIcon::setIcon(QIcon &icon)
         };
         QImage* image = new QImage(icon.pixmap(64, 64).toImage());
         if (image->format() != QImage::Format_RGBA8888_Premultiplied)
-        *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
+            *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
         GdkPixbuf* pixbuf = gdk_pixbuf_new_from_data(image->bits(), GDK_COLORSPACE_RGB, image->hasAlphaChannel(),
                                  8, image->width(), image->height(),
                                  image->bytesPerLine(), callbackFreeImage, image);
+        if(!pixbuf) // If pixbuf initialization fails, it won't trigger the callback.
```

Comment:
This looks bullshit to me. If pixbuf initialization fails, the callback should be still called. Are you sure about this?